uk.co.compucorp.civicrm.hrleaveandabsences/views/my-leave/components/my-leave-request.html

```diff
@@ -0,0 +1,177 @@
+                           uib-datepicker-popup
+                           show-button-bar="false"
+                           datepicker-options="{startingDay: 1,showWeeks: false}"
+                           ng-model="modal.leaveRequest.fromDate"
+                           is-open="modal.leaveRequest.showDatePickerFrom"/>
+                    <span class="input-group-btn">
+                        <button type="button" class="btn btn-default fa fa-calendar"
+                                ng-click="modal.leaveRequest.showDatePickerFrom = true"></button>
+                      </span>
+                  </div>
+                </div>
+                <div class="col-xs-4">
+                  <div class="chr_custom-select chr_custom-select--full">
+                    <select>
+                      <option value="1" selected>1/2 - PM</option>
+                      <option value="2">1/2 - AM</option>
+                    </select>
+                  </div>
+                </div>
+              </div>
+            </div>
+          </div>
+          <div class="row">
+            <div class="col-xs-12 form-horizontal">
+              <div class="form-group">
+                <label class="col-xs-4 control-label">To Date
+                  <i class="fa fa-asterisk"></i>
+                </label>
+                <div class="col-xs-4">
+                  <div class="input-group chr-datepicker">
+                    <input type="text"
+                           class="form-control"
+                           uib-datepicker-popup
+                           datepicker-options="{startingDay: 1,showWeeks: false}"
+                           show-button-bar="false"
+                           ng-model="modal.leaveRequest.toDate"
+                           is-open="modal.leaveRequest.showDatePickerTo"/>
+                    <span class="input-group-btn">
+                        <button type="button" class="btn btn-default fa fa-calendar"
+                                ng-click="modal.leaveRequest.showDatePickerTo = true"></button>
+                      </span>
+                  </div>
+                </div>
+                <div class="col-xs-4">
+                  <div class="chr_custom-select chr_custom-select--full">
+                    <select>
+                      <option value="1" selected>1/2 - PM</option>
+                      <option value="2">1/2 - AM</option>
+                    </select>
+                  </div>
+                </div>
+              </div>
+            </div>
+          </div>
+          <hr/>
+          <div class="row form-group">
+            <div class="col-xs-4">Opening Balance</div>
+            <div class="col-xs-8">17.0</div>
+          </div>
+          <div class="row form-group text-primary pointer"
+               ng-click="modal.leaveRequest.isChangeExpanded = !modal.leaveRequest.isChangeExpanded">
+            <div class="col-xs-4">
+              <i class="fa chr-inline-icon"
```

Comment:
Don't agree with this custom class as well. Just to be clear: a `chr_{xyz}` (please note that the prefix is `chr_`, not `chr-`) should be created only when you have to do something *from scratch*

Here what you are really doing is 1) having a font awesome icon smaller than the default size 2) giving layout rules to the icon

This is something that you could achieve easily if you used the `.chr_leave-request-form` class (or similar) that i mentioned above. You could have then something like:
```scss
.chr_leave-request-form__legend {
  
  .fa {

  }
}
```